recover-id-object/scripts/Dockerfile

```diff
@@ -1,12 +1,10 @@
-ARG build_image=rust:1.70-buster
-ARG base_image=debian:buster-slim
+ARG build_image=rust:1.70-bookworm
+ARG base_image=debian:bookworm-slim
 FROM ${build_image} AS build
 
 WORKDIR /usr/app/recover-id-object
-
-COPY recover-id-object .
-COPY deps /usr/app/deps
-RUN cargo build --release --locked
+COPY . .
```

Comment:
This might perform decent enough now, but since we are copying the entire repo as build context it might not scale as we add more and more.
Also this is not very nice when running locally, since it would include `target` and everything... I think

Reply:
Okay, we will see how it scales I guess